live-rates/src/adapter.ts

```diff
@@ -0,0 +1,33 @@
+import { Execute } from '@chainlink/types'
+import { Requester, Validator } from '@chainlink/external-adapter'
+
+const customParams = {
+  symbol: ['base', 'from', 'symbol', 'rate'],
+  to: false,
+}
+
+export const execute: Execute = async (input) => {
+  const validator = new Validator(input, customParams)
+  if (validator.error) throw validator.error
+
+  const jobRunID = validator.validated.id
+  const url = 'https://live-rates.com/api/price'
+  const symbol = validator.validated.data.symbol.toUpperCase()
+  const to = (validator.validated.data.to || '').toUpperCase()
+  const rate = symbol + to
+  const key = process.env.API_KEY
```

Comment:
If this is a required API_KEY it would be good to use our internal util for that here:
import { util } from '@chainlink/ea-bootstrap'
```
const util.getRandomRequiredEnv('API_KEY')

```
This util looks for the `API_KEY` environment variable and throws if it isn't provided. It also allows multiple API_KEYs to be provided that it will cycle through.